Src/IronPython.Modules/_ctypes/Structure.cs

```diff
@@ -50,6 +50,10 @@ public abstract class _Structure : CData {
                 INativeType nativeType = NativeType;
                 StructType st = (StructType)nativeType;
 
+                if (args is null) { // None passed as a sole argument
```

Comment:
Interesting, `None` gives `null` here? Would adding the `NotNoneAttribute` solve this?

Reply:
Yes, `None` gives `null`. This is the consequence that the DLR allows to bind to the `param object[]` array directly (just like C#). We have discussed it before (IronLanguages/dlr#249). Although then I was arguing against it, I now think that for params arrays it will be better to keep the behaviour as it is now.

But indeed, `[NotNone]` prevents it from happening, pushing `null` into the params array. Good insight! It is an elegant solution.